tests/foreman/cli/test_import.py

```diff
@@ -960,6 +962,221 @@ def test_import_repo_recovery(self, test_data):
                 Repository.info({'id': record['sat6']}).return_code, 0
             )
 
+    @data(*gen_import_cv_data())
+    def test_import_cv_default(self, test_data):
+        """@test: Import and enable all Content Views from the default data set
+        (predefined source)
+
+        @feature: Import Enable Content View
+        @assert: 3 Content Views imported and enabled
+
+        """
+        # randomize the values for orgs and repos
+        tmp_dir = self.default_dataset[0]
+        files = dict(self.default_dataset[1])
+        files['content-views'] = os.path.join(
+            tmp_dir, 'exports/CHANNELS/export.csv'
```

Comment:
When skimming through this code, it's not obvious that there are two arguments. Please place each argument on a separate line.

```python
        files['content-views'] = os.path.join(
            tmp_dir,
            'exports/CHANNELS/export.csv',
        )
```